### PR TITLE
Bylaws refresh - Avoid gendered language and change Secretary to Vice President

### PR DESCRIPTION
**Pull request by peterjc**
The board has discussed avoiding gendered language, and subsuming the Secretary role (with historical gendered associations) into a vice or deputy president role. This pull request attempts to put this in writing.

~~This pull request would also allow Electronic ballots can stay open up to 24 hours for global participation. This change does not actually help with getting quorum, just makes it more explicit how absentee board members might still be able to vote (even though they can't be counted toward being quorate).~~

~~(Alternatively, we might introduce proxy votes to enable voting from board members whose timezone makes attending live impractical.)~~

HTML and PDF update to follow once markdown changes are agreed in principle.

### CODE REVIEW

File: OBF Bylaws.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ ARTICLE II - MEETINGS
 ARTICLE III - BOARD OF DIRECTORS
 ----------------------------
 
-1.	The Board of Directors shall consist of President, Secretary, Treasurer, and additional Board Members at Large. The number of Board Members at Large is decided by majority vote of the current Board of Directors and may be proposed by either the general membership or a Board Member.
+1.	The Board of Directors shall consist of President, Deputy President, Treasurer, and additional Board Members at Large. The number of Board Members at Large is decided by majority vote of the current Board of Directors and may be proposed by either the general membership or a Board Member.
```
Comment (nlharris) [inline OBF Bylaws.md:44]: Maybe I missed this decision, but why "Deputy President"? Isn't Vice President a more widely used term? (And a vice president similarly is empowered to act on behalf of the president if the president is not available.)

**Review by nlharris — COMMENTED** [review]
Other than my uncertainty about the "Deputy President" role, the changes look good.

I think this document would be easier to read if there were line breaks between sub-items (which there are when you edit the md, but not when you look at the file on github).

File: OBF Bylaws.md (hunk 1 above)
Comment (cjfields) [reply OBF Bylaws.md:44]: +1 for vice president.

Comment (peterjc) [reply OBF Bylaws.md:44]: I'll change it. The terms seem to be used interchangeably, perhaps with a UK/US bias. We have a Deputy Prime Minister, you have a Vice President ;)

File: OBF Bylaws.md [hunk 2]
```diff
@@ -83,19 +85,19 @@ ARTICLE IV - NOMINATIONS AND ELECTIONS
...
     b.	Nominees or appointees for Officer positions must have been voting members of the OBF for a minimum of one year, be in good standing, and have previously served on the Board of Directors for a minimum of one year.
     c.	The Board of Directors may waive qualification requirements at its discretion by unanimous consent of attending Directors, provided the quorum is met. If the membership requirement is waived for a nominee, the nominee automatically becomes a member upon election, and must meet all requirements to be in good standing within 4 (four) weeks after election. 
 7.	_Removed_.
-8.	Paper ballots shall be counted by 2 (two) active members of the OBF selected by the President. Electronic ballots will be counted electronically.
+8.	Paper ballots shall be counted during the meeting by 2 (two) active members of the OBF selected by the President. Electronic ballots will be counted electronically once each ballot is closed, which may be up to 24 hours later to facilitate participation by absentee board members in other time zones (who would not be counted toward quorum).
```
Comment (peterjc) [inline OBF Bylaws.md:100]: One might argue the old wording is sufficiently broad that we don't need this change?

Comment (peterjc) [reply OBF Bylaws.md:100]: Hilmar agrees on #101 that the old policy wording allows this anyway, and this only restricts us. I'll revert it...

**Review by cjfields — APPROVED** [review]

**Review by kipkurui — APPROVED** [review]

**Review by nlharris — APPROVED** [review]

**Comment by peterjc** [comment]
We did not vote on this as https://www.open-bio.org/2022/09/29/obf-public-board-meeting-2022-10-21/ but deferred to the next public meeting.

**Comment by nlharris** [comment]
We made the minor changes in the bylaws to avoid gendered language, so the task in the title of this ticket has been accomplished. Can we close this and open new tickets as needed for any additional bylaws changes we want to propose?

**Comment by cjfields** [comment]
...
> [Edited to add: though actually it looks like there's a PR linked that still needs public approval?]

Yep, it looks like we need at least one more approval.

**Comment by hlapp** [comment]
I thought we said we wanted to package this up with other pending bylaws changes and put it up for a membership vote?

**Comment by peterjc** [comment]
That's my recollection too, see draft minutes of the public board meeting at #106.

**Review by nlharris — APPROVED** [review]
Can we update the title of this PR to mention the change of title from Secretary to Vice President?

Also, weren't we going to add to the Vice President's duties that they can stand in for the President when the President is unavailable?

**Comment by hlapp** [comment]
> weren't we going to add to the Vice President's duties that they can stand in for the President

We did: https://github.com/OBF/obf-docs/pull/102/files#diff-ba35f0d8561bfd58f0eaf56ebd60812e4be370c2051227bb3e589bcede402aadR66

File: OBF Bylaws.md [hunk 3]
```diff
@@ -54,14 +54,16 @@ ARTICLE III - BOARD OF DIRECTORS
...
+        (3)	Interpret and rule on procedural matters at all meetings
         (4)	Appoint special committees and committee members as necessary
         (5)	Assume responsibility for the preparation of agendas for General Membership meeting
-    b.	Secretary
+    b.	Vice President
         (1)	Record the proceedings of all meetings and maintain a file of those minutes
         (2)	Maintain a file of all OBF business
         (3)	Prepare meeting minutes for Board meetings (as defined in Article II)
         (4)	Maintain a master file of all reports, minutes, correspondence, and documents
+        (5) Considered to be the OBF's Secretary if required by external parties
```
Comment (hlapp) [inline OBF Bylaws.md:65]: Use whitespace indentation consistent with preceding rows:
```suggestion
        (5)	Considered to be the OBF's Secretary if required by external parties
```

File: OBF Bylaws.md [hunk 4]
```diff
@@ -54,14 +54,16 @@ ARTICLE III - BOARD OF DIRECTORS
...
+        (6) Deputise for the President, taking on their role when the President is unavailable
```
Comment (hlapp) [inline OBF Bylaws.md:66]: Use whitespace consistent with preceding rows:
```suggestion
        (6)	Deputise for the President, taking on their role when the President is unavailable
```

**Review by hlapp — APPROVED** [review]
See inline edits, but they only affect whitespace formatting, not substance or even language.

**Comment by nlharris** [comment]
> > weren't we going to add to the Vice President's duties that they can stand in for the President
> 
> We did: https://github.com/OBF/obf-docs/pull/102/files#diff-ba35f0d8561bfd58f0eaf56ebd60812e4be370c2051227bb3e589bcede402aadR66

Oh, weird, it didn't show me that line in the diff. Ok!

**Review by nlharris — APPROVED** [review]

**Comment by hlapp** [comment]
> Can we update the title of this PR to mention the change of title from Secretary to Vice President?

Yes, and I did that now.

**Review by HLWiencko — APPROVED** [review]

**Review by gedankenstuecke — APPROVED** [review]

**Review by kipkurui — APPROVED** [review]

**Pull request merged**